tests/charm++/dynamic_array_expansion/dynamical_insertion.ci

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
```

Comment:
Presumably we want these tests to fall under the normal Charm++ license, so we should probably remove these blocks from all the files. The original authors of the tests have already implicitly agreed to this, but it's perhaps worth double checking with them that they're willing to contribute this under our usual license.